test/e2e/config/docker-dev.yaml

```diff
@@ -47,7 +47,7 @@ providers:
     - old: "imagePullPolicy: Always"
       new: "imagePullPolicy: IfNotPresent"
     - old: "--enable-leader-election"
-      new: "--enable-leader-election=false"
+      new: "--enable-leader-election=false\n        - --feature-gates=ClusterResourceSet=true"
```

Comment:
question (given that I see the change applied only to `docker-dev.yaml` and not to `docker-ci.yaml`): are you planning to enable e2e tests for this feature in a separated PR?

Reply:
Added it now. Thanks.